src/Microsoft.AspNet.Razor/CodeGenerators/GeneratedTagHelperContext.cs

```diff
@@ -29,10 +29,10 @@ public GeneratedTagHelperContext()
             MarkAsHtmlEncodedMethodName = "Html.Raw";
             StartTagHelperWritingScopeMethodName = "StartTagHelperWritingScope";
             EndTagHelperWritingScopeMethodName = "EndTagHelperWritingScope";
-            RunnerTypeName = "TagHelperRunner";
-            ScopeManagerTypeName = "TagHelperScopeManager";
-            ExecutionContextTypeName = "TagHelperExecutionContext";
-            TagHelperContentTypeName = "TagHelperContent";
+            RunnerTypeName = "Microsoft.AspNet.Razor.Runtime.TagHelperRunner";
+            ScopeManagerTypeName = "Microsoft.AspNet.Razor.Runtime.TagHelperScopeManager";
+            ExecutionContextTypeName = "Microsoft.AspNet.Razor.Runtime.TagHelperExecutionContext";
+            TagHelperContentTypeName = "Microsoft.AspNet.Razor.TagHelperContent";
```

Comment:
Well, the type names need to be valid for a C# context so the lines are a bit blurry. And CodeDOM isn't the perfect model to follow :smiling_imp: but :ok: as long as all type names in this context are treated consistently.